### ISSUE TITLE
Event str raises typeError with empty origin

### ISSUE THREAD

**Issue by d-chambers** [obspy/obspy#2119]
Setup: Ubuntu 16.04, python 3.6, current Obspy master

The following code raises a TypeError:

```python
from obspy.core.event import Event, Origin
print(Event(origins=[Origin()]))
```
This should be a simple fix. The `short_str` method of the `Event` class needs to account for the case when origin times, latitudes, and/or longitudes are None.

**Comment by megies** [comment]
This doesnt look like extremely high priority to me, so I'll drop it into 1.2.1 for now.. feel free to fix it for 1.1.1 if you want, though.

**Comment by krischer** [comment]
Fixed in #2127 and #2132
